pandas/tests/test_graphics.py

```diff
@@ -991,6 +991,18 @@ def test_plot(self):
         df = DataFrame({'x': [1, 2], 'y': [3, 4]})
         with tm.assertRaises(TypeError):
             df.plot(kind='line', blarg=True)
+        try:
+            df.plot(color = ['red', 'black'], style = ['-', '--'])
+            df['x'].plot(color = 'red', style = '-')
+        except:
+            self.fail("Calling 'plot()' on a dataframe/series and passing both 'color' and 'style' arguments should be allowed if there is no color symbol in the style string(s)")
+        try:
+            df.plot(color = ['red', 'black'], style = ['k-', 'r--'])
```

Comment:
This one should raise and exception, so you can use the `tm.assertRaises` context manager

```python
with tm.assertRaises(ValueError):
     df.plot(color = ['red', 'black'], style = ['k-', 'r--'])
```

so that the test passes if a ValueError is raised, and fails if it doesn't.